libs/community/langchain_community/cache.py

```diff
@@ -1365,6 +1366,11 @@ def get_md5(input_string: str) -> str:
 ASTRA_DB_CACHE_DEFAULT_COLLECTION_NAME = "langchain_astradb_cache"
 
 
+@deprecated(
+    since="0.1.23",
```

Comment:
Using the langchain-core version as was done for the Pinecone VectorStore: https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/pinecone.py#L46
but shouldn't we use the main langchain version (or community) ?

Reply:
Changed to 0.0.22 as per https://github.com/langchain-ai/langchain/pull/17732#discussion_r1500096177